coderd/database/modelmethods.go

```diff
@@ -683,9 +683,16 @@ func ConvertWorkspaceRows(rows []GetWorkspacesRow) []Workspace {
 			NextStartAt:             r.NextStartAt,
 			TaskID:                  r.TaskID,
 		}
+
+		if err := workspaces[i].UserACL.Scan(r.UserACL); err != nil {
+			return nil, xerrors.Errorf("scan user ACL %q: %w", r.UserACL, err)
+		}
+		if err := workspaces[i].GroupACL.Scan(r.GroupACL); err != nil {
+			return nil, xerrors.Errorf("scan group ACL %q: %w", r.GroupACL, err)
+		}
```

Comment:
These fields were just missing before?

Another missing field I see is  `OwnerName: r.OwnerName,`

Reply:
Done.

However it looks like it isn't used:
https://github.com/coder/coder/blob/geokat/internal-858-sharing-info-in-workspaces-response/coderd/workspaces.go#L2678

But I added it for completeness.